### PR TITLE
[WIP]test

### PR DESCRIPTION
**Pull request by ToIshikawa**
test pull request

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+
```
Comment (yj2222) [inline app/controllers/application_controller.rb]: test_comments

Comment (ToIshikawa) [reply app/controllers/application_controller.rb]: TKS

**Comment by ToIshikawa** [comment]
test_comment2

**Comment by yj2222** [comment]
LGTM

**Pull request merged**